### ISSUE TITLE
Binary Mask

### ISSUE THREAD

**Issue by Goru1890** [backseason/PoolNet#63]
It is possible to get back a binary mask (only 0 and 255 values) instead of a grayscale?

**Comment by backseason** [comment]
You can either threshold the grayscale map or replace the last conv layer to have 2-channel output and the loss function from BCE to CE (re-train required and not tested).

**Comment by Goru1890** [comment]
Will the result be the same using either of the two options?

**Comment by backseason** [comment]
No, I suggest you try the 1st option first as the 2nd one requires re-training the model and some of the hyper-parameters may need to be adjusted accordingly.

**Comment by Goru1890** [comment]
I tried the 1st option, but the results are not always according to the original (tried different thresholds), sometimes the mask is bigger than the grayscale mask and sometimes smaller. 

I tried also the 2nd option, but this is to complicated for me, I'm just a novice.

Thanks for your help anyway
